src/pages/profile/edit/index.tsx

```diff
@@ -0,0 +1,241 @@
+import {
+  ChevronRight,
+  AccountCircleOutlined,
+  SettingsOutlined,
+  GppGoodOutlined,
+} from '@mui/icons-material';
+import { Box, Button, Link, Paper, Stack, Typography, StackProps } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { styled } from '@mui/material/styles';
+import { useAppSelector } from '../../../hooks/redux-hooks';
+import useLogoutMutation from '../../../api/mutations/account/useLogoutMutation';
+import { MuiStylesObject } from '../../../types/utility.types';
+
+const CustomStack = styled(Stack)<StackProps>(({ theme }) => ({
+  height: '100%',
+  gap: '1rem',
+  [theme.breakpoints.up('xs')]: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    borderBottom: '1px solid',
+    borderColor: '#EBEBEB',
+    padding: '1rem 0',
+  },
+  [theme.breakpoints.up('md')]: {
+    flexDirection: 'column',
+    alignItems: 'stretch',
+    border: 'none',
+    padding: '1rem',
+    boxShadow: '0px 6px 16px rgba(0, 0, 0, 0.12)',
+    borderRadius: '0.75rem',
+  },
+}));
+
+const styles = {
+  userImage: {
+    width: {
+      xs: '9rem',
+      md: '12rem',
+    },
+    height: {
+      xs: '9rem',
+      md: '12rem',
+    },
+    bgcolor: 'primary.main',
+    borderRadius: '50%',
+    flexShrink: 0,
+    objectFit: 'cover',
+  },
+} satisfies MuiStylesObject;
```

Comment:
Yes, I think satisfies does great job here.

MuiStylesObject is a custom type which is Record<string, SxProps<typeof theme>>.

Since key of the object is typed just as a string, we are not getting type checking on selecting a property (styles.userImage) when using MuiStylesObject type just as 'styles:MuiStylesObject'. Meanwhile, with 'satisfies' we can get type checking.